### DISCUSSION TITLE
Is there any way to set pollInterval for health check query?

### DISCUSSION THREAD

**Discussion by ezhuikova** [apollographql/apollo-server#7077] [General]
We upgrated apollo server to 4 version and noticed health check query spams a lot to logs.
How to reduce the frequency of calling this health check request?

async function startApolloServer() {
  const server = new ApolloServer<MyContext>({ typeDefs, resolvers });
  const { url } = await startStandaloneServer(server, {
    context: async ({ req }) => ({ token: req.headers.token }),
    listen: { port: 4000 },
  });
  console.log(`🚀  Server ready at ${url}`);
}

One more question how to restrict external queries that new playground(apollo sandbox) calls

**Comment by glasser** [comment]
What health check are you referring to? Apollo Server 4 no longer has a health check feature: https://www.apollographql.com/docs/apollo-server/migration/#health-checks

**Reply by ezhuikova** [reply]
I'm reffering [GraphQL-level health checks](https://www.apollographql.com/docs/apollo-server/monitoring/health-checks/#graphql-level-health-checks)

сurl --header "apollo-require-preflight: true" localhost:4000/graphql?query=%7B__typename%7D
{"data":{"__typename":"Query"}}

**Reply by glasser** [reply]
OK, then I guess the question is what logs are you referring to? Apollo Server itself doesn't have any built-in request logging.

**Reply by ezhuikova** [reply]
sorry, my bad! There is custom logger that logs few validation messages on one health check query. Thank you!

What about my second question?  Is there a way to restrict external queries that new playground(apollo sandbox) calls.

**Reply by glasser** [reply]
Sounds like you should change your logger to not log for health checks by looking at the URL or some header or something.

I don't understand your second question, sorry!
